lib/puma/binder.rb

```diff
@@ -273,6 +283,25 @@ def parse(binds, logger, log_msg = 'Listening')
       end
     end
 
+    def localhost_authority
+      @localhost_authority ||= Localhost::Authority.fetch if defined?(Localhost::Authority) && !defined?(JRUBY_VERSION)
+    end
+
+    def localhost_authority_context
+     if !localhost_authority.nil?
+        local_certificates_path = File.expand_path("~/.localhost")
```

Comment:
This should be after the `else`

Reply:
@nateberkopec  I am not sure what do you mean by this. Can i know where to put it?